### PR TITLE
added astral-portfolio-page!

### PR DESCRIPTION
**Pull request by Tamanna-S**
hello, i have made this a guided project by adding spec.yml and 6 steps that contains challenges.yml and instructions.md.
check this out and let me know if that's okay.

Working on Astral Portfolio Project #33

### CODE REVIEW

File: astral-portfolio-page/step0/Instructions.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Hello Folks!!
+
+so, here goes the instructions for step0 :
```
Comment (Pranavtechie) [inline astral-portfolio-page/step0/Instructions.md:3]: @Tamanna-S , you can remove the `step0` text from the instructions page. We do not share this numbers with the users anywhere

File: astral-portfolio-page/step1/instructions.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Welcome in the next step!
```
Comment (Pranavtechie) [inline astral-portfolio-page/step1/instructions.md:1]: The Name of the file `instructions` is in lowercase, can you change it to `Instructions.md`

Comment (Pranavtechie) [inline astral-portfolio-page/step1/instructions.md:1]: Grammer : `In this step, you'll be building, doing .... `

File: astral-portfolio-page/step2/instructions.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+huh! champ welcome in the styling part.
```
Comment (Pranavtechie) [inline astral-portfolio-page/step2/instructions.md:1]: renaming to capital I `Instructions.md`

File: astral-portfolio-page/step3/instructions.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+welcome in the part of styling the contact page!
```
Comment (Pranavtechie) [inline astral-portfolio-page/step3/instructions.md:1]: Capital I in the file name

File: astral-portfolio-page/step6/instructions.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+finally the last step and you have completed a nice portfolio project!!!
```
Comment (Pranavtechie) [inline astral-portfolio-page/step6/instructions.md:1]: @Tamanna-S, the first Instructions.md contains the capital letter you need to rename the rest of the files. 

Remember that changing a single letter doesn't get recognized on git. For that you have to rename the file name with an extra charact and then commit the changes made, and then go back and update the file name to `Instructions.md`

**Review by Pranavtechie — CHANGES_REQUESTED** [review]

**Comment by Tamanna-S** [comment]
really sorry for the mistakes fixing them all!!

**Comment by Tamanna-S** [comment]
hey please i messed up with the branch may i open a new pr and close this?

**Pull request closed without merging**